ext/curl/interface.c

```diff
@@ -1598,12 +1514,65 @@ PHP_FUNCTION(curl_copy_handle)
 }
 /* }}} */
 
+static bool php_curl_set_callable_handler(zend_fcall_info_cache *const handler_fcc, zval *callable, bool is_array_config, const char *option_name)
+{
+	if (ZEND_FCC_INITIALIZED(*handler_fcc)) {
+		zend_fcc_dtor(handler_fcc);
+		handler_fcc->function_handler = NULL;
```

Comment:
I guess this isn't necessary anymore now we know that zend_fcc_dtor overwrites the data with the empty fcc.